source/common/network/io_socket_handle_impl.cc

```diff
@@ -196,7 +214,7 @@ Address::InstanceConstSharedPtr maybeGetDstAddressFromHeader(const cmsghdr& cmsg
     return getAddressFromSockAddrOrDie(ss, sizeof(sockaddr_in), fd);
   }
   return nullptr;
-}
+} // namespace Network
```

Comment:
I think this is a mistake. Same below. Revert?

Reply:
you mean the comment `// namespace Network`?

Reply:
and the comments `// namespace <name>` in general in the file

Reply:
This comment is on the trailing } of a function which I think is an accident.

Reply:
I see thanks! I think my `clang-format` messed up the parsing and added them and I never realized it.

Reply:
fixed in c1d4d36567caae65137fe36f66fe188c0622533c